src/Clipper.jl

```diff
@@ -534,6 +542,10 @@ function Base.length(c::__ClipperPolyTree)
     @cxx c->Total()
 end
 
+function child_count(c::__ClipperPolyTree)
+    @cxx c->ChildCount()
+end
+
```

Comment:
I've removed `getindex` from this PR. If we later decide we want it we can add it back in in another PR.